doc/reference/configuration/configuration_reference.rst

```diff
@@ -2400,7 +2441,12 @@ The ``process`` defines configuration parameters of the Tarantool process in the
 
 .. confval:: process.background
 
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
     Run the server as a background task.
```

Comment:
We can call it 'daemonization' (daemon process, daemonized process and so on). This is a well know term.